packages/inspector/src/internal-inspect.ts

```
@@ -0,0 +1,210 @@
+import inspectNumber from "./types/number";
+import inspectObject from "./types/object";
+import inspectPromise from "./types/promise";
+import inspectRegExp from "./types/regexp";
+import inspectSet from "./types/set";
+import inspectString from "./types/string";
+import inspectSymbol from "./types/symbol";
+import inspectTypedArray from "./types/typed-array";
+import { getIndent } from "./utils/indent";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+const constructorMap = new WeakMap<Function, Inspect>();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)